### PR TITLE
Removed CentOS bare metal manual config documentation

### PR DESCRIPTION
**Pull request by rothgar**
Documentation uses a users repo for packages (not official) which has old package versions (1.3.0) and contains old tools (kube-version-change) that are confusing to users. It also doesn't set up kube-dns or kube-proxy to make a fully functioning multi-node cluster.

After working with a friend for days who was trying to set up a cluster I found out he was using this guide which was causing many problems and conflicting information from other parts of the documentation. At first I was going to try and fix this guide but I think we should point people to the [kubeadm](http://kubernetes.io/docs/getting-started-guides/kubeadm/) documentation as that has steps for a CentOS installation.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2185)

### CODE REVIEW

File: docs/getting-started-guides/centos/centos_manual_config.md [hunk 1]
```diff
@@ -1,236 +0,0 @@
----
-assignees:
-- thockin
-title: CentOS
----
-
```
Comment (devin-donnelly) [inline docs/getting-started-guides/centos/centos_manual_config.md]: For what it's worth, I think we should post a notice that this document is out of date and deprecated, along with a redirect or a new link that users should follow. I think that's better than simply deleting the file outright. We can still remove it from the table of contents and add an exception to the TOC check, as it is considered deprecated.

**Comment by devin-donnelly** [comment]
Some feedback on this; see review. Also, there's a merge conflict in the GS Guides Index Page.

File: docs/getting-started-guides/centos/centos_manual_config.md (hunk 1 above)
Comment (rothgar) [inline docs/getting-started-guides/centos/centos_manual_config.md]: While I would normally agree with that I have been working with new users for the past few weeks and the sheer amount of content and old/broken/miss information is causing more confusion than I think it's worth to keep an old version of a document around and redirect to where someone should be looking. This makes google searching and searching within the documentation itself more confusing as users are less likely to read a disclaimer if the page exists and shows up at the top of their search.
I think having CentOS documentation is valuable to have but this document needs a complete re-write to be up to date with best practices and get an up to date cluster running (including kube-proxy, dns, addons). I also think a lot of that effort is being spent on kubeadm so ideally people would find that documentation before anything else.

**Comment by chenopis** [comment]
@rothgar ping. Just checking in on the status of this PR. Is there anything else you need?

**Comment by rothgar** [comment]
I need to fix the conflict but I don't think we agreed on if the document should be completely removed or just put a notice on the document that it's depreciated and remove it from the TOC.

**Comment by chenopis** [comment]
@devin-donnelly what say ye?

**Comment by rothgar** [comment]
Here's another example of someone last month having trouble with this existing document.

http://serverfault.com/questions/822600/kubernetes-dns-missing-missingclusterdns

**Comment by chenopis** [comment]
@rothgar Based on the process of other document migrations we are currently performing, I believe the best course of action would be:

Step 1: Post a deprecation notice on the page to be migrated. The deprecation notice should state why the content is being removed and that the page will be removed from the site nav (ToC) and search index in two weeks, and from Kubernetes.io entirely two weeks after that.

Step 2 (2 weeks after posting notice): Remove the page from the TOCs and from the site search index. We will not delete the actual content at this time, to preserve inbound links and bookmarks. Those readers should at least hit the deprecation notice and be warned to update their inbounds.

Step 3 (2 weeks after removing from nav): Remove the page from Kubernetes.io entirely.

**Comment by rothgar** [comment]
Is there a standard banner to notify readers of the depreciation? Should I submit 3 pull requests for each stage and then just have them merged at specific dates?

**Comment by chenopis** [comment]
There isn't a standard banner yet for regular deprecated content. However, for the docs/user-guide/ migration, which does involve some deprecation, we created a [user-guide-content-moved.md](https://github.com/kubernetes/kubernetes.github.io/blob/master/_includes/user-guide-content-moved.md) file in `/_includes` as the standard banner for that, then we just include it in a page using this syntax:

`{% include user-guide-content-moved.md %}`

There are some examples of the banner in the wild in the Guides / Container and Pods section, e.g. [Running Your First Containers](https://kubernetes.io/docs/user-guide/simple-nginx/).

You can do something similar. The text in the banner could be something like, "This content in the Kubernetes docs is being deprecated for the following reason(s):"

Then, in the file after the include, add a paragraph or list of bullet points stating the reason(s). Finally, at the end, add a statement like, "This page will be removed from the site on <deprecation_date>."

Yes, each stage should be done as a separate pull request and merged on specific dates. I can set some reminders for myself to help merge them for you.

**Comment by chenopis** [comment]
@rothgar I'm going to close this PR for now since there are some merge conflicts and it's getting stale. Feel free to reopen it, and let us know how you want to proceed w/ the deprecation.

**Comment by rothgar** [comment]
Fair enough. I've had no time to work on this and brought it up in sig onprem today as a candidate to be re-written or someone else can submit new PRs to get rid of it.

**Comment by chenopis** [comment]
No worries. When you find a spare cycle to work on this, you can always ping me to help you. Cheers.

**Pull request closed without merging**